### PR TITLE
RAINCATCH-1218 - application guide

### PR DESCRIPTION
**Pull request by wtrocki**
Still in progress - creating for visibility.

### CODE REVIEW

File: docs/workforce-management-framework/topics/ref-server-application.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- mongodb installed and running on port 27017
+URL controlled by `process.env.MONGO_CONNECTION_URL` and `config-{env}.json` file.
+- redis installed and running on port 6379
+URL controlled by `process.env.REDIS_CONNECTION_URL` and `config-{env}.json` file.
+
+> Note: If you running RainCatcher solution as part of RHMAP platform this requirements will be satisfied automatically and you do not need change any values in configuration.
+
+== Configuration
+
+Demo offers different configuration profiles depending on target application deployment.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-server-application.adoc]: Try not to use demo

File: docs/workforce-management-framework/topics/ref-server-application.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+All configuration parameters are documented directly in javascript file.
+Developers building their solution on top of the RainCatcher are free to extend configuration for their own needs.
+
+== Structure
+
+`modules` - folder contains sample implementation for RainCatcher integrations like sync, security.
+
+`modules` -
+
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-server-application.adoc]: WIP

File: docs/workforce-management-framework/topics/ref-portal.adocs [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Repository structure
+
+### `src/app`
+Main folder containing the logic for the application. The app is mainly concerned in providing the setup and configuration values for the RainCatcher and angularjs modules that are used by it as well as for the [FeedHenry JavaScript SDK](https://github.com/feedhenry/fh-js-sdk).
+
+#### `src/app/services`
+Contains angularjs services that are utilized by the RainCatcher modules to retrieve data. The app has the opportunity to provide an implementation that utilizes any technoligy for data retrieval, including static fixtures for testing.
+
+This demo application utilizes an [`fh-sync`](https://github.com/feedhenry/fh-sync)-based service, which would allow the client to work offline for extended amounts of time, syncronizing with the server whenever a data connection becomes available again.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-portal.adocs]: demo again

File: docs/workforce-management-framework/topics/ref-portal.adocs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+#### `src/app/sass`
+Contains app-wide [Sass](http://sass-lang.com/) files that are compiled by the build pipeline.
+
+### `www/`
+Generated directory containing the output files of the local build process.
+
+This directory's contents are what is served by the local development web server.
+
+## Running The Demo Solution Locally
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/ref-portal.adocs]: demo again

**Comment by wtrocki** [comment]
@austincunningham - Ready for review.

**Comment by wtrocki** [comment]
ping @paolobueno - you may find this useful.

File: docs/workforce-management-framework/topics/ref-mobile.adoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
+
+[id='{context}-ref-portal']
+= {PROJECT_NAME} Mobile application
```
Comment (paolobueno) [inline docs/workforce-management-framework/topics/ref-mobile.adoc]: wrong ref-portal here

Comment (austincunningham) [reply docs/workforce-management-framework/topics/ref-mobile.adoc]: mobile

**Pull request merged**